src/service/chain_rewardable_entities.proto

```diff
@@ -0,0 +1,177 @@
+// TODO: Design for different key formats - track "archetype" of key ... helium
+// vs. solana, network prefixes etc.
+// TODO: Consider OTHER Rewardable Entities (REs) in the future
+// TODO: Signature STUFF in req's and reports
+
+syntax = "proto3";
+
+package helium.chain_rewardable_entities;
+
+enum pub_key_format {
+  pub_key_format_UNKNOWN = 0; // Unknown format, should not be used
+  pub_key_format_B58 = 1;     // Base58 format
+  pub_key_format_UTF8 = 2;    // UTF8 format
+}
+
+message pub_key {
+  string value = 1;
```

Comment:
Not actually sure what this is representing? An entity key? A solana pubkey? String and not bytes here seems odd